### ISSUE TITLE
Pretrain-model get higher accuracy than backbone.

### ISSUE THREAD

**Issue by byhwhite** [koncle/CoraNet#11]
Dear author:
Appreciate to your amazing work first.
I run your code on pancreas dataset, pretrain model could get 76.62% accuracy on test set. While the TABLE.Ⅴ in paper shows that the V-net accuracy is 69.96%, and I'm confused about the difference. Hope you could help me!
Looking forward to your reply!

**Comment by koncle** [comment]
The accuracy reported in TABLE. V is the performance of the original version of V-Net. We have optimized it for our method, which gives higher accuracy for the backbone. We also test the accuracy of [UA-MT](https://github.com/koncle/CoraNet/blob/master/train_UA_MT.py) with the same setting and does not get higher accuracy.

**Comment by wtkszzz** [comment]
> The accuracy reported in TABLE. V is the performance of the original version of V-Net. We have optimized it for our method, which gives higher accuracy for the backbone. We also test the accuracy of [UA-MT](https://github.com/koncle/CoraNet/blob/master/train_UA_MT.py) with the same setting and does not get higher accuracy.

Dear author, can you give details of the optimisation on the pancreas dataset, I can only train out the performance of the original version of Vnet using the pancreas dataset, thank you very much!

**Comment by GuozhengSui** [comment]
I would like to consult you about the missing module in the Train UA mt.py file about Train_Vnet.py.
How do you run it successfully？
